mypy/checkexpr.py

```diff
@@ -1686,6 +1686,8 @@ def visit_ellipsis(self, e: EllipsisExpr) -> Type:
 
     def visit_op_expr(self, e: OpExpr) -> Type:
         """Type check a binary operator expression."""
+        if e.op == 'in':
```

Comment:
Does this mean that we won't check type errors within the subexpressions of the in expressions?

Concretely, does mypy still report an error on code like `1 in ([1] + ['x'])`?